astronomy/time_scales_body.hpp

```diff
@@ -102,7 +105,7 @@ constexpr std::array<int, (2018 - 1972) * 2 + 1> leap_seconds = {{
     +1, +0,  // 2015
     +0, +1,  // 2016
     +0, +0,  // 2017
-    +0,      // 2018
+    +0, +0,  // 2018
```

Comment:
Let's do this in a separate pull request and update the generated files at the same time.